validator/src/lib.rs

```diff
@@ -13,7 +13,15 @@ pub use validation::must_match::validate_must_match;
 pub use validation::non_control_character::validate_non_control_character;
 #[cfg(feature = "phone")]
 pub use validation::phone::validate_phone;
+pub use validation::range::validate_range_generic;
+
+#[deprecated(
+    since = "0.12.0",
+    note = "Please use the validate_range_generic function instead"
+)]
+#[allow(deprecated)]
```

Comment:
I don't think anyone is using that crate without the derive, we can probably have the breaking change without deprecation.

Reply:
Alright, that makes some things easier :)